stack.yaml

```diff
@@ -32,6 +32,11 @@ resolver: lts-19.6
 #  subdirs:
 #  - auto-update
 #  - wai
+
+# FIXME: A lot of dependency contraints are not satisfied when building, after
```

Comment:
I think these comments should be removed before merging.

Reply:
Yes, they inded should be.